content/en/docs/contribute/new-content/contributing-content.md

```diff
@@ -0,0 +1,267 @@
+---
+title: Opening a pull request
+slug: new-content
+content_template: templates/concept
+weight: 10
+card:
+  name: contribute
+  weight: 40
+---
+
+{{% capture overview %}}
+
+{{< note >}}
+**Kubernetes code developers**: If you are documenting a new feature for an
+upcoming Kubernetes release, your process is a bit different. See
+[Document a new feature](/docs/contribute/new-content/new-features/) for
+process guidelines and information about deadlines.
+{{< /note >}}
+
+To contribute new content pages or improve existing content pages, open a _pull request_ (PR). Make sure you follow all the requirements in the [Before you begin](#before-you-begin) section.
```

Comment:
nit: Sometimes the contribute is one new line,
To contribute new content or improve existing content

nit: Make sure your follow all of the requirements in the [This link text does not describe where the link is located -- definitely not on his page, /docs/contribute/new-content/overview/#before-you-begin]